### PR TITLE
Add random shuffle option to slideshow

### PR DESCRIPTION
**Pull request by dosman711**
This adds 2 new options for slideshow, `rnd-slideshow [time]` and `rnd-slideshow-<region> [time]`. Also some cleanup to help text to help it all fit.

Supersedes PR #45

### CODE REVIEW

**Comment by gvela024** [comment]
Could you update the `README.md` as well?

**Comment by dosman711** [comment]
Of course, sorry about that!

File: README.md [hunk 1]
```diff
@@ -75,15 +75,17 @@ Parameters:
...
+    regions                       -   List all the available regions.
+    extra                         -   List all the Pokemon from the 'Extra' folder.
+    random                        -   Change the terminal background to a random Pokemon.
+    random-<region>               -   Change the terminal background to a random Pokemon from the specified region.
+    slideshow [time]              -   Iterate through each Pokemon. Optional time (in seconds) between Pokemon.
+    slideshow-<region> [time]     -   Iterate through each Pokemon in the specified region. Optional time (in seconds) between Pokemon.
+    rnd-slideshow [time]          -   Iterate through each Pokemon in a random order. Optional time (in seconds) between Pokemon.
+    rnd-slideshow-<region> [time] -   Iterate through each Pokemon in the specified region in a random order. Optional time (in seconds) between Pokemon.
+    clear | disable | off         -   Clear the Pokemon in the terminal.
+    help                          -   Display this menu.
```
Comment (gvela024) [inline README.md]: :heart_eyes:

**Comment by LazoCoder** [comment]
```
$ pokemon slideshow-johto 2
Invalid command specified.
Type "help" to see all the commands.
```

**Comment by dosman711** [comment]
My mistake, I tested it, made a change, and failed to test again. Sorry about that!

**Comment by LazoCoder** [comment]
I like the new help menu.

**Pull request merged**